src/tools/react-native.ts

```diff
@@ -7,3 +9,45 @@ export const isAndroidInstalled = (toolbox: GluegunToolbox): boolean => {
 
   return Boolean(hasAndroid)
 }
+
+type Params = {
```

Comment:
is `Params` too generic of a name?

Reply:
I'll be honest, I was trying to keep it short so it wouldn't word-wrap the function definition. :laughing: